etc/insights-client.spec

```diff
@@ -136,8 +135,11 @@ if ! [ -d "/var/lib/insights" ]; then
 mkdir -m 644 /var/lib/insights
 fi
 
-# symlink the motd file
-ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
+#Link motd into place. Only do so if motd file doesn't exist
+if ! [ -L /etc/motd.d/insights-client ]; then
+    mkdir -p /etc/motd.d
+    ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
```

Comment:
The `-f` should be redundant now with the `-L` check from above, but harmless.

Reply:
Actually, I believe this is weird/wrong. The documentation says "by pointing /etc/motd.d/insights-client at an empty file", so you woud be fully entitled to do "touch /etc/motd.d/insights-client" and expect it to be and stay quiet. But that will fail the `-L` check, and this `ln -f` will remove the file and replace it with a symlink. So I'd suggest to drop the `-f` (for safety) and fix the check to

    if [ ! -e /etc/motd.d/insights-client -a ! -L /etc/motd.d/insights-client ]; then

Reply:
Hmmm. While it may be surprising ```touch /etc/motd.d/insights-client``` would work ... as the symlink would always redirect that somewhere.

In addition the documentation in the manual page states:

```
To change or  remove  this  message, edit the /etc/insights-client/insights-client.motd file.
```

I'm not sure this is a bug and have @gravitypriest doing gymnastics to redo the build, yet again.

Reply:
For the record, this isn't a dealbreaker, just a tiny glitch. No need to re-spin.